src/Symfony/Component/DependencyInjection/EnvVarProcessor.php

```diff
@@ -77,7 +77,7 @@ public function getEnv(string $prefix, string $name, \Closure $getEnv): mixed
             }
 
             if (!isset($array[$key]) && !\array_key_exists($key, $array)) {
-                throw new EnvNotFoundException(sprintf('Key "%s" not found in %s (resolved from "%s").', $key, json_encode($array), $next));
+                throw new EnvNotFoundException(sprintf('Key "%s" not found in "%s" (resolved from "%s").', $key, json_encode($array), $next));
```

Comment:
this one should be reverted, it's a false-positive from fabbot

Reply:
done